go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -69,6 +70,18 @@ func start(t *testing.T) (utils.MySQLCompare, func()) {
 	}
 }
 
+func TestAggrWithLimit(t *testing.T) {
+	utils.SkipIfBinaryIsBelowVersion(t, 20, "vtgate")
```

Comment:
We should merge https://github.com/vitessio/vitess/pull/16266 first for the upgrade/downgrade test.
```suggestion
```